plugins/kafka/kafka.ecllib

```diff
@@ -212,9 +212,10 @@ EXPORT Consumer(VARSTRING topic, VARSTRING brokers = 'localhost', VARSTRING cons
         END;
 
         // Set the offset for each partition on each node
-        result := PROJECT
+        result := NORMALIZE
             (
-                distOffsets,
+                NOFOLD(distOffsets),
```

Comment:
So I did try the minimal fix to the PROJECT, which I thought was simply adding NOFOLD, but I never managed to make the DISTRIBUTE stick.  Where should the NOFOLD be applied?  I tried it in its current location, around DISTRIBUTE, around the attribute being distributed, and maybe one or two other places.